### PR TITLE
Add Python 3.9-dev job to Travis-CI

### PR DESCRIPTION
**Pull request by atugushev**
Python 3.9 release is on October 5, 2020.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -60,7 +63,7 @@ jobs:
           tags: true
           repo: jazzband/pip-tools
   allow_failures:
-    - env: PIP=master
+    - python: 3.8-dev
```
Comment (atugushev) [inline .travis.yml:66]:
```suggestion
    - python: 3.9-dev
```

**Comment by webknjaz** [comment]
FYI you can add this to GHA: https://github.com/ansible/ansible-lint/pull/836/files#diff-436179b9415ec7fbbc24dfce8f289167R61-R66.

**Comment by atugushev** [comment]
@webknjaz thanks! I'll close this PR since we are moving towards GitHub Actions. I'll open new PR with python 3.9 on GHA then.

**Comment by atugushev** [comment]
Went to #1169.

**Pull request closed without merging**